src/www/firewall_rules.php

```diff
@@ -373,7 +373,7 @@
 <?php
                 $interface_has_rules = false;
                 foreach ($a_filter as $i => $filterent):
-                if ( (isset($filterent['interface']) && $filterent['interface'] == $selected_if) ||
+                if ( (isset($filterent['interface']) && in_array($selected_if, explode(',', $filterent['interface']))) ||
```

Comment:
If no interfaces are set for a floating rule, all interface match, that currently doesn't happen